doc/classes/File.xml

```diff
@@ -279,7 +280,7 @@
 			<argument index="0" name="enable" type="bool">
 			</argument>
 			<description>
-				Set whether to swap the endianness of the file. Enable this if you're dealing with files written in big endian machines.
+				Set whehter to swap the endianness of the file. Enable this if you're dealing with files written in big endian machines.
```

Comment:
Got a misspelling here. Also, try using the same boolean rule here:

    If [code]true[/code] the file's endianness is swapped. Use this if...